package.json

```diff
@@ -29,7 +29,8 @@
     "prettier": "./prettier-commits.sh",
     "prettier-on": "prettier --write",
     "test": "jest",
-    "db:setup": "psql \"$DATABASE_URL?sslmode=prefer\" -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
+    "db:setup": "dotenv npm run db:setup:run",
+    "db:setup:run": "psql \"$DATABASE_URL?sslmode=prefer\" -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
```

Comment:
@Maascha had the problem, that the database did not exist yet. Should we add a command to create it as well?

Reply:
Hm, one anyways has to create a user and grant them permissions (either to create a db or on the created db)

Reply:
hmm... that's true.
Let's see if anyone is running into a similar issue